### PR TITLE
API docs: format and test all resources

### PR DESCRIPTION
**Pull request by andie787**
### Summary of changes

- put Machines API doc in the new format
- text API calls and fix some things
- combine or split some section
- create all new partials with no formatting
- merge in Volumes and Apps resource branches

### Preview

https://aa-docs.fly.dev/docs/machines/api-machines-resource/
https://aa-docs.fly.dev/docs/machines/api-volumes-resource/
https://aa-docs.fly.dev/docs/machines/api-apps-resource/

### CODE REVIEW

File: machines/api-machines-resource.html.markerb [hunk 1]
```diff
@@ -57,85 +149,234 @@ You can configure the Machine characteristics, like its CPU and memory. You can
...
+      description: 'Bad Request'
+    ) %>
+    <% api_info.add_info(
+      name: '408',
+      description: 'Request Timeout'
+    ) %>
+    <%= render(api_info) %>
+  </div>
+  <div> 
+    <%= render(CodeToggleComponent.new(badge: 'GET', title: '/apps/{app\_name}/machines/{machines\_id}/wait')) do |component| %>
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:318]:
```suggestion
    <%= render(CodeToggleComponent.new(badge: 'GET', title: '/apps/{app\_name}/machines/{machine\_id}/wait')) do |component| %>
```

File: machines/api-machines-resource.html.markerb [hunk 2]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
+      description: 'Created'
+    ) %>
+    <% api_info.add_info(
+      name: '409',
+      description: 'Conflict'
+    ) %>
+    <%= render(api_info) %>
+  </div>
+  <div> 
+    <%= render(CodeToggleComponent.new(badge: 'POST', title: '/apps/{app\_name}/machines/{machines\_id}/lease')) do |component| %>
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:645]:
```suggestion
    <%= render(CodeToggleComponent.new(badge: 'POST', title: '/apps/{app\_name}/machines/{machine\_id}/lease')) do |component| %>
```

File: machines/api-machines-resource.html.markerb [hunk 3]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
+    ) %>
+    <% api_info = ApiInfoComponent.new(
+      heading: 'Responses',
+      name: '404',
+      description: 'Not Found'
+    ) %>
+    <%= render(api_info) %>
+  </div>
+  <div> 
+    <%= render(CodeToggleComponent.new(badge: 'GET', title: '/apps/{app\_name}/machines/{machines\_id}/lease')) do |component| %>
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:701]:
```suggestion
    <%= render(CodeToggleComponent.new(badge: 'GET', title: '/apps/{app\_name}/machines/{machine\_id}/lease')) do |component| %>
```

File: machines/api-machines-resource.html.markerb [hunk 4]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
+      description: 'OK'
+    ) %>
+    <% api_info.add_info(
+      name: '400',
+      description: 'Bad Request'
+    ) %>
+    <%= render(api_info) %>
+  </div>
+  <div> 
+    <%= render(CodeToggleComponent.new(badge: 'DELETE', title: '/apps/{app\_name}/machines/{machines\_id}/lease')) do |component| %>
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:758]:
```suggestion
    <%= render(CodeToggleComponent.new(badge: 'DELETE', title: '/apps/{app\_name}/machines/{machine\_id}/lease')) do |component| %>
```

**Review by catflydotio — CHANGES_REQUESTED** [review]

File: machines/api-machines-resource.html.markerb (hunk 1 above)
Comment (andie787) [reply machines/api-machines-resource.html.markerb]: fixed everywhere!

File: machines/api-volumes-resource.html.markerb [hunk 5]
```diff
@@ -190,29 +304,50 @@ Status: 200 OK
...
+      <% component.with_json do %>
+      no body
+      <% end %>
+      <% component.with_json_table do %>
+      | Property | Type | Required | Description |
+      | --- | --- | --- | --- |
+      | no body |  |  |  |
+      <% end %>
+    <% end %>
+    <%= render(CodeSampleComponent.new(title: 'Status: 200 OK', language: 'json')) do %>
```
Comment (catflydotio) [inline machines/api-volumes-resource.html.markerb:350]:
```suggestion
    <%= render(CodeSampleComponent.new(title: 'Status: 200 OK – Example response', language: 'json')) do %>
```

File: machines/api-volumes-resource.html.markerb [hunk 6]
```diff
@@ -232,36 +367,59 @@ Status: 200 OK
...
-
-```json
+      <% end %>
+      <% component.with_json_table do %>
+      | Property | Type | Required | Description |
+      | --- | --- | --- | --- |
+      | size_gb | integer | yes | The size in GB to make the volume. |
+      <% end %>
+    <% end %>
+    <%= render(CodeSampleComponent.new(title: 'Status: 200 OK', language: 'json')) do %>
```
Comment (catflydotio) [inline machines/api-volumes-resource.html.markerb:416]:
```suggestion
    <%= render(CodeSampleComponent.new(title: 'Status: 200 OK – Example response', language: 'json')) do %>
```

File: machines/api-apps-resource.html.markerb [hunk 7]
```diff
@@ -33,31 +104,173 @@ v4   37.16.9.52 global 7s ago
...
+
+
+## List apps in an organization
+
+`GET /apps?org_slug=`
+
+<div class="api-section" data-exclude-render>
+  <div>
+    <% api_info = ApiInfoComponent.new(
+      heading: 'Path parameters',
```
Comment (catflydotio) [inline machines/api-apps-resource.html.markerb:226]:
```suggestion
      heading: 'Query parameters',
```

**Review by catflydotio — COMMENTED** [review]
After a skim of apps and volumes

File: machines/api-machines-resource.html.markerb [hunk 8]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
+      name: 'app_name',
+      type: 'string',
+      required: true,
+      description: 'The name of the Fly App the Machine belongs to.'
+    ) %>
+    <% api_info.add_info(
+      name: 'machine_id',
+      type: 'string',
+      required: true,
+      description: 'The ID of the Machine to get.'
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:480]:
```suggestion
      description: 'The ID of the Machine to stop.'
```

File: machines/api-machines-resource.html.markerb [hunk 9]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:531]:
```suggestion
      description: 'The ID of the Machine to start.'
```

File: machines/api-machines-resource.html.markerb [hunk 10]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
+      name: 'app_name',
+      type: 'string',
+      required: true,
+      description: 'The name of the Fly App the Machine belongs to.'
+    ) %>
+    <% api_info.add_info(
+      name: 'machine_id',
+      type: 'string',
+      required: true,
+      description: 'The ID of the Machine to wait for.'
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:580]:
```suggestion
      description: 'The ID of the Machine to delete.'
```

File: machines/api-machines-resource.html.markerb [hunk 11]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:630]:
```suggestion
      description: 'The ID of the Machine to create a lease for.'
```

File: machines/api-machines-resource.html.markerb [hunk 12]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:685]:
```suggestion
      description: 'The ID of the Machine to retrieve any existing lease for.'
```

File: machines/api-machines-resource.html.markerb [hunk 13]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:743]:
```suggestion
      description: 'The ID of the Machine to release any existing lease for.'
```

File: machines/api-machines-resource.html.markerb [hunk 14]
```diff
@@ -147,119 +388,490 @@ This is, in particular, how you would update the running image of a Machine (whe
...
+      name: 'app_name',
+      type: 'string',
+      required: true,
+      description: 'The name of the Fly App the Machine belongs to.'
+    ) %>
+    <% api_info.add_info(
+      name: 'machine_id',
+      type: 'string',
+      required: true,
+      description: 'The ID of the Machine to get.'
```
Comment (catflydotio) [inline machines/api-machines-resource.html.markerb:798]:
```suggestion
      description: 'The ID of the Machine to cordon or uncordon.'
```

**Pull request merged**